### DISCUSSION TITLE
Which noise scheduler is generally better?

### DISCUSSION THREAD

**Discussion by YanJiaHuan** [huggingface/diffusers#9284] [Q&A]
Hi everyone, sorry for asking such a question... But i am new to noise scheduler. Currently, one of my projects is to try different noise scheduler modules in a diffusion model. 
Can anyone give me some ideas? Or is there any research work or blog proposing any findings on choosing the optimal noise schedulers in different situations?
Thanks!

**Comment by tolgacangoz** [comment]
Hi @YanJiaHuan. I haven't finished reading all these yet, but I think these are must-reads:

- Paper: [On the Importance of Noise Scheduling for Diffusion Models](https://huggingface.co/papers/2301.10972)
- Blog post: [Noise schedules considered harmful](https://sander.ai/2024/06/14/noise-schedules.html)
	- [EDM paper video presentation](https://www.youtube.com/watch?v=T0Qxzf0eaio)

**Reply by YanJiaHuan** [reply]
Thanks for your sharing.

**Comment by dumoulina** [comment]
Same question. From what I saw, some have their advantages and weaknesses so there is no best scheduler.
Most people answer that you should test them, but mathematically this i not an answer.
I would recommend trying sde-dpmsolver++ as of now.

Also I still don't understand the relation between the scheduler used for training and the one used for inference.
I did not found any best practice about scheduler for training.
